Chinese-Manual/spec_CN.md

```diff
@@ -0,0 +1,482 @@
+# CloudEvents 核心规范-  1.0.1 版本
+
+Declaration： This manual aims to provide a fast and brief introduction of CloudEvents
+in Chinese for people who are new to CloudEvents.
+Most of the content is translated from the original English version.
+It is strongly recommended to read the English version if you find anything lost in translation.
+
+## 摘要
+
+CloudEvents 是一个用于定义事件格式的供应商中立规范。
+
+## 目录
+
+- [概览](#概览)
+- [符号和术语](#符号和术语)
+- [上下文属性](#上下文属性)
+- [事件数据](#事件数据)
+- [大小限制](#大小限制)
+- [隐私与安全](#隐私与安全)
+- [示例](#示例)
+
+## 概览
+
+事件(Events)在现代系统中无处不在。但不同的事件生产者往往用不同的规范来描述自己的事件。
+
+对事件的统一描述的匮乏意味着开发者必须不断重新学习如何消费不同定义的事件。
+它同样限制了那些用来帮助事件数据完成 跨环境传输的库(如SDKs)，
+工具(如事件路由器)和基础设施(如事件追踪系统)的发展。
+总体来看，这种匮乏严重阻碍了事件数据的可移植性和生产力。
+
+CloudEvents是一个以通用格式来描述事件数据的标准。它提供了事件在服务、平台和系统中的互操作性。
+
+事件格式指定如何使用某些编码格式序列化一个 CloudEvent。
+支持那些编码且兼容 CloudEvents 的实现必须遵守相应事件格式中指定的编码规则。
+所有实现都必须支持 [JSON 格式](json-format.md)。
+
+有关规范背后的历史、开发和设计原理等更多信息，
+请参阅 CloudEvents  [入门文档](primer_CN.md)。
```

Comment:
These links are broken when I clicked them. Not sure if the issue is specific to me.